Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -137,6 +137,7 @@
   #define DOGLCD
   #define IS_U8GLIB_ST7920 1
   #define IS_ULTIPANEL 1
+  #define ENCODER_PULSES_PER_STEP 2
```

Comment:
You had some `REPRAPWORLD_GRAPHICAL_LCD` pins earlier, but they are not included in the new PR. Did you intend to remove them?

Reply:
I don' t think I had them in this place. Marlinui_dogm has been updated to allow hardware SPI, maybe that is what you mean?